### PR TITLE
update DB doc

### PR DESCRIPTION
**Pull request by StarfallProjects**

### CODE REVIEW

File: docs/hosting/architecture/database-structure.md [hunk 1]
```diff
@@ -19,20 +19,36 @@ To inspect the n8n database, you can use [DBeaver](https://dbeaver.io){:target=_
...
+Stores the history of a SAML connection. [TODO - confirm]
+
 ### credentials_entity
 
 Stores the credentials used to authenticate with integrations.
 
-### eventdestinations_entity
+### event_destinations
 
 Enterprise only.
```
Comment (Joffcom) [inline docs/hosting/architecture/database-structure.md:36]: This table does still exist, This part may not be needed.

File: docs/hosting/architecture/database-structure.md [hunk 2]
```diff
@@ -19,20 +19,36 @@ To inspect the n8n database, you can use [DBeaver](https://dbeaver.io){:target=_
...
 
 Contains the destination configurations for [Log streaming](/log-streaming/).
 
+### execution_data
+
+Links executions to workflow data. [TODO - confirm]
```
Comment (Joffcom) [inline docs/hosting/architecture/database-structure.md:42]: This is the actual execution data and contains the workflow at the time of running and the execution data

File: docs/hosting/architecture/database-structure.md (hunk 1 above)
Comment (StarfallProjects) [reply docs/hosting/architecture/database-structure.md:36]: I can't see it with DBeaver? It's not in the diagram it generated either?

Comment (Joffcom) [reply docs/hosting/architecture/database-structure.md:36]: @StarfallProjects that is interesting, I can see it in my Postgres database. I would be surprised if we made a table when the license is populated 🤔

Comment (StarfallProjects) [reply docs/hosting/architecture/database-structure.md]: Let us never speak of this again

**Review by Joffcom — APPROVED** [review]

**Pull request merged**